test/unit/config/test__base.py

```diff
@@ -360,4 +360,4 @@ def test_containerization_unsafe_write_setting(tmp_path, runtime, mocker):
         'podman': 1,
```

Comment:
It only seems reasonable to expect a string like this
```suggestion
        'podman': '1',
```